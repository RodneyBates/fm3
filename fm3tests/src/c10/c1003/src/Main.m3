MODULE Main EXPORTS MMff , Grbl  

(* C test, identifier references. *)

; TYPE P = REF R

; TYPE R
  = RECORD
      F1 : INTEGER 
    ; F2 : CHAR
    END 

; BEGIN
  END Main
.

