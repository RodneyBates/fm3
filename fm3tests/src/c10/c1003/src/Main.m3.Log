[32mINFO:  [0mMain.m3 Pass 1 output file Main.m3.FM3Pass1 has 564 bytes.
[32mINFO:  [0mMain.m3 Patch stack Main.m3.FM3Patch peak size = 72 bytes.
[32mINFO:  [0mMain.m3 Pass 2 output file Main.m3.FM3Pass2 has 305 bytes.
