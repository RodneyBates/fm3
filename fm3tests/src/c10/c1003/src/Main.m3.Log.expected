[31mERROR: [0mMain.m3[1,20] Unable to locate exported source file MMff.i3
       Search directories for source files are:
       "."
[31mERROR: [0mMain.m3[1,27] Unable to locate exported source file Grbl.i3
